drivers/hmis/app/graphql/types/forms/enums/pick_list_type.rb

```diff
@@ -49,6 +49,7 @@ class Forms::Enums::PickListType < Types::BaseEnum
     value 'STAFF_ASSIGNMENT_RELATIONSHIPS', 'Staff Assignment Relationships'
     value 'USERS', 'User accounts. Deprecated in favor of AUDITABLE_USERS'
     value 'ELIGIBLE_STAFF_ASSIGNMENT_USERS', 'Current users who are eligible for staff assignment'
+    value 'ELIGIBLE_REFERRAL_STEP_ASSIGNMENT_USERS', 'Current users who can be assigned to referral steps in the specified project'
```

Comment:
```suggestion
    value 'ELIGIBLE_STAFF_ASSIGNMENT_USERS', 'Users who are eligible for staff assignment'
    value 'ELIGIBLE_REFERRAL_STEP_ASSIGNMENT_USERS', 'Current users who can be assigned to referral steps in the specified project'
```
"current users" is confusing to me, we use the term "current user" for the _currently logged in user_. If you are referring to active users we can say active users, but I think its probably sufficient to say "users" because its implied that only active users would be eligible